packages/stores/src/impls/hashmap.rs

```diff
@@ -0,0 +1,289 @@
+use std::{
```

Comment:
maybe a btreemap too? a sorted set with `.top()` or `.last()` would be nice. `VecDeque` potentially?